roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -1,13 +1,19 @@
 ---
+- name: Load lib_openshift modules
+  include_role:
+    name: lib_openshift
+
 - name: Create passthrough route for docker-registry
-  command: >
-    {{ openshift.common.client_binary }} create route passthrough
-    --service docker-registry
-    --config={{ openshift_hosted_kubeconfig }}
-    -n default
+  oc_route:
+    name: passthrough
+    namespace: default
+    service_name: docker-registry
+    kubeconfig: "{{ openshift_hosted_kubeconfig }}"
+    state: present
   register: create_docker_registry_route
-  changed_when: "'already exists' not in create_docker_registry_route.stderr"
+  changed_when: "create_docker_registry_route.changed == True"
```

Comment:
Do we need this `register` and `changed_when` clause?  It seems this would be by default from the module to returned `changed: true`